package.json

```diff
@@ -18,7 +18,7 @@
     "split2": "4.1.0"
   },
   "engines": {
-    "node": ">=8",
+    "node": ">=12",
```

Comment:
This isn't really breaking given Pino 7.x is only guaranteed against 12+

https://github.com/pinojs/pino/blob/v7.9.2/docs/lts.md#schedule